monitor/README.md

```diff
@@ -0,0 +1,38 @@
+# Cilium node monitor
+
+The node monitor provides an API for reading the events from the BPF datapath.
+When the process `cilium-node-monitor` is started it handles new connections to
+`$RuntimePath/monitor.sock`. Users of the API are expected to read the
+[Meta][0] and [Payload][1]. Since the payload can vary in size to make decoding
```

Comment:
Perhaps phrase this as:
` Users of the API are expected to read the Meta and Payload structs (encoded in gob).`

It's not a big deal and it should be obvious that they are structs if you click through.